notebook/Dockerfile

```diff
@@ -12,4 +12,5 @@ RUN chmod +x /usr/bin/prepare.sh
 RUN mkdir /opt/app
 USER $NB_USER
 
-CMD ["bash", "-c", "/usr/bin/prepare.sh && start.sh jupyter lab"]
+ENTRYPOINT ["tini", "--", "bash", "/usr/bin/prepare.sh"]
```

Comment:
So we need to download it. There is a conda-forge package. Alternatively can use prebuilt binaries from the author.

Reply:
Also I guess we still need `-c` here.

Reply:
I think that it is in the base-notebook image https://github.com/jupyter/docker-stacks/blob/master/base-notebook/Dockerfile#L92

Reply:
Ah sorry, we are using the base-notebook as a parent image. Missed that. NVM then.

Reply:
We might be able to get away without `bash` here actually.